Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py

```diff
@@ -103,16 +141,37 @@ def parse_plea_type(plea_text: str):
     :param plea_text: Plea text in case dockets
     :return: 'Not Guilty', 'Guilty', 'Nolo Contendere', or None.
     """
-    if 'NOT' in plea_text:
-        plea = 'Not Guilty'
-    elif 'GUILTY' in plea_text:
-        plea = 'Guilty'
-    elif 'NOLO' in plea_text:
-        # Nolo-Contendere (no contest) plea
-        plea = 'Nolo Contendere'
-    else:
-        plea = None
-    return plea
+    try:
+        if 'NOT' in plea_text:
+            plea = 'Not Guilty'
+        elif 'GUILTY' in plea_text:
+            plea = 'Guilty'
+        elif 'NOLO' in plea_text:
+            # Nolo-Contendere (no contest) plea
+            plea = 'Nolo Contendere'
+        else:
+            plea = None
+        return plea
+    except TypeError:
+        return None
+
+
+def parse_name(fullname_text: str) -> (str, str, str):
+    """
+    Parses the first, middle and last name from a full name.
+    :param fullname_text: Defendant's fullname as a String
+    :return: (FirstName, MiddleName, LastName)
+    """
+    if fullname_text == '' or fullname_text is None:
```

Comment:
A simpler check that's very pythonic is `if not fullname_text:`, which covers both of your cases.